lib/ansible/module_utils/basic.py

```diff
@@ -2185,7 +2185,19 @@ def log(self, msg, log_args=None):
                 for arg in log_args:
                     journal_args.append((arg.upper(), str(log_args[arg])))
                 try:
-                    journal.send(u"%s %s" % (module, journal_msg), **dict(journal_args))
+                    if HAS_SYSLOG:
```

Comment:
this is a backwards incompatible change, the syslog setting has always ONLY affected machines not using systemd's journal

Reply:
@bcoca thank you for the follow-up!
If this approach has a problem with backwords compatibility, I think that it would be nice if we could choose the logging method in ansible.cfg(syslogd or journald).

Reply:
Thank you for your review!!
I noticed that LOG_LOCAL0 to 7 are not defined in systemd.journal, therefore I referenced those variables from syslog module. but you are right.
I'll modify this to define those variables in basic.py and use it and avoid to use syslog module.
Please let me know if I made misunderstanding!